transform/mattermost-analytics/models/intermediate/sales/int_charges_filtered_onprem.sql

```diff
@@ -0,0 +1,50 @@
+{{config({
+    "materialized": 'table',
+    "unique_key":"charge_id",
+  })
+}}
+
+select
+    s.customer_id,
+    s.subscription_id,
+    s.renewed_from_subscription_id,
+    s.license_id,
+    i.invoice_id,
+    i.charge_id,
+    s.start_at,
+    s.ended_at,
+    p.name as plan_name,
+    s.license_start_at,
+    s.license_end_at,
+    s.actual_renewal_at,
+    ili.quantity,
+    ili.quantity - LAG(ili.quantity) OVER (PARTITION BY s.customer_id ORDER BY i.created_at) as seat_difference,
+    datediff(day, lag(license_end_at) OVER (PARTITION BY s.customer_id ORDER BY i.created_at), license_start_at) as days_since_previous_license_end,
+    row_number() over(partition by s.customer_id order by i.created_at) as charge_order,
+    case
+        when s.renewed_from_subscription_id is not null then true
+        when days_since_previous_license_end < 60 then true
+        else false end
+    as is_renewal,
```

Comment:
@johndavidlugtu to help validate/confirm the renewal logic?

Reply:
I'm also still working on that. Seems that there's some additional cases (i.e. trials) that we should consider.